sidecar/RELEASING.md

```diff
@@ -0,0 +1,41 @@
+Releasing Cook Executor
+=======================
+
+Cook Sidecar is released on [PyPI](https://pypi.org/project/cook-executor/)
+
+Prerequisites
+-------------
+Ensure you can build the sidecar following the instructions in README.md
+
+Install `twine`:
+```bash
+pip3 install twine
+```
+
+Update version number
+------------
+Look at setup.py, and edit the version number which may reference another python variable to be edited.
+
+Test Release
+------------
+Since PyPI does not allow modifying releases, it can be useful to test a release using their test instance.
+```bash
+rm -rf dist/*
+python3 setup.py sdist bdist_wheel
```

Comment:
Not sure if this is worth mentioning or if this is just some Python distribution basics that I missed, but `bdist_wheel` isn‘t available unless you `pip install wheel`